repo/packages/A/arangodb/0/config.json

```diff
@@ -0,0 +1,86 @@
+{
+  "type": "object",
+  "properties": {
+
+    "mesos/master": {
+      "description": "The URL of the Mesos master. The format is a comma-delimited list of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard URLs like http://localhost are also acceptable.",
+      "type": "string",
+      "default": "zk://master.mesos:2181/mesos"
+    },
+
+    "arangodb/id": {
+      "description": "Identifier for the cluster.",
+      "type": "string",
+      "default": "arangodb"
+    },
+
+    "arangodb/zk": {
+      "description": "ZooKeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path. Please note that the id is appended to the path.",
+      "type": "string",
+      "default": "zk://master.mesos:2181/arangodb"
+    },
+
+    "arangodb/webui-host": {
+      "description": "For debugging: use this host for WEBUI request.",
+      "type": "string",
+      "default": ""
+    },
+
+    "arangodb/framework-port": {
+      "description": "Framework port, use '0' to let Mesos select a port for you.",
+      "type": "integer",
+      "default": 0
+    },
+
+    "arangodb/volume-path": {
+      "description": "Path for persistent data (until persistent volumes are available).",
+      "type": "string",
+      "default": "/tmp"
+    },
+
+    "arangodb/mode": {
+      "description": "Mode for framework, possible values: \"standalone\", \"cluster\"",
+      "type": "string",
+      "default": "cluster"
+    },
+
+    "arangodb/minimal-resources-agent": {
+      "description": "Minimal mesos resources for an agent",
+      "type": "string",
+      "default": "mem(*):512;cpus(*):0.2;disk(*):512"
```

Comment:
I do not know. I have not found a specification or documentation for mesos minimal resource specifications anywhere, so all I did is guesswork. I understood that the star here means that the framework will accept offers for any role. If one would put (arangodb) instead it would accept only offers made for the role arangodb. Is this right?